cmake/ConfigSGX.cmake

```diff
@@ -5,32 +5,31 @@
 # get arch information
 execute_process(COMMAND getconf LONG_BIT OUTPUT_VARIABLE VAR_LONG_BIT)
 
-# Set variables for common SGX variables.
-# if they are defined in the environment, take them from there, otherwise
-# set them to the usual defaults.
+# Check for the existence of common required SGX variables,
+# and fail if these are not defined.
```

Comment:
```suggestion
# and fail if these are not defined. Note global build defaults are defined `../config.mk`.
```

Reply:
yes! -- I suppose the `co-author` line is not recognized as a sign-off. Disappointing!